pkg/embed/embed.go

```diff
@@ -7,9 +7,14 @@ import (
 
 	"github.com/YourFin/binappend"
 	"github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/code-ready/crc/pkg/crc/version"
 )
 
 func openEmbeddedFile(executablePath, embedName string) (io.ReadCloser, error) {
+	if version.IsInstaller() {
+		panic("should no longer happen")
+	}
```

Comment:
@cfergeau do we really need those panic , shouldn't we just send error back to caller function.

Reply:
This `panic` commit was not meant to be pushed to this PR, dunno how it ended up there :)